### PR TITLE
ADR for Sirius Integration

### LINKED ISSUE #53: ADR: Integration with Sirius

### PR DESCRIPTION
**Pull request by johnnolan**
Adds a ADR around our intent to integrate with Sirius (or rather the database that Sirius runs on)

Fixes #53

### CODE REVIEW

File: src/adr/articles/0008-integration-with-sirius.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+### Interrogate your data decisions
+
+When we integrate with Sirius, it is important to ensure we only have access to the data we need and nothing more.
+
+If the part of the service is responsible for adding data and validating that data, then the integration point should for example only allow adding of data and getting a confirmation of valid or not from the integration. We should not be returning data and comparing it on the Modernising side.
+
+### Decision
+
+We should integrate with the Sirius database and not directly with the case management system.
```
Comment (gregtyler) [inline src/adr/articles/0008-integration-with-sirius.md:97]: Should we be integrating with the Sirius (Public?) API? Rather than the database itself

Comment (johnnolan) [reply src/adr/articles/0008-integration-with-sirius.md]: You are right, thank you, my meaning is completely lost in that sentence

File: src/adr/articles/0008-integration-with-sirius.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---
+layout: layouts/adr.njk
+title: "Integration with Sirius"
```
Comment (mattmachell) [inline src/adr/articles/0008-integration-with-sirius.md:3]: I've had a think about this and I think what tripped me up is the framing as "integration with Sirius", which is not really what we're doing. We're integrating with the OPG's data that Sirius happens to be the default window on and current owner of the data schema. Both of which are things that can and will change over time.

Comment (johnnolan) [reply src/adr/articles/0008-integration-with-sirius.md:3]: Yeah I agree, as I started writing it became really clear this wasn't the actual thing we wanted to do and is confusing. I think I cover this in the article however the title now doesn't match this. Do you think changing the title to `Integration with OPG data` is better and adjust content where needed to match that title?

Comment (mattmachell) [reply src/adr/articles/0008-integration-with-sirius.md:3]: I think the reframing might help. I wonder if really it's about the end to end visibility of data for business processes.

Is the decision really, Modernise and Sirius both need to access the same data and for it to be a single source of truth? Is that source a new LPA Data service that sits initially as part of Sirius but is eventually independent of the two?

Comment (johnnolan) [reply src/adr/articles/0008-integration-with-sirius.md]: Have adjusted the content to what I think better reflects our current setup and context

**Review by hawx — APPROVED** [review]

**Pull request merged**